### PR TITLE
Add Verification Level command

### LINKED ISSUE #255: Server Verification Level Settings
It would be good if GearBot had a command where we could change the verification level from low to double table-flip in case of any serious situations such as spam bots, raiding, etc.

### PR DESCRIPTION
**Pull request by Kitty-Therapist**
This should close #255, I've tested out the verification level command and it is functioning properly as it should be.

It does works with low, medium, high, and extreme.

I've thrown the logging for MOD_ACTIONS for the time being as AE is planning on changing it up soon.

### CODE REVIEW

File: GearBot/Cogs/Moderation.py [hunk 1]
```diff
@@ -424,6 +425,26 @@ def _can_act(action, ctx, user, check_bot=True):
...
+        """verification_help"""
+        v1 = discord.VerificationLevel.__members__.get(level.lower())
+        if v1 is not None:
+            if ctx.guild.verification_level != v1:
+                try:
+                    await ctx.guild.edit(verification_level=v1, reason=reason)
+                except discord.Forbidden:
+                    await MessageUtils.send_to(ctx, 'NO', "verification_no_perms")
+                    return
+                else:
```
Comment (trilleplay) [inline GearBot/Cogs/Moderation.py]: You're doing a return and a else, one of them is enough

File: GearBot/Cogs/Moderation.py [hunk 2]
```diff
@@ -424,6 +425,26 @@ def _can_act(action, ctx, user, check_bot=True):
...
+                    if reason is None:
+                        reason = ''
```
Comment (trilleplay) [inline GearBot/Cogs/Moderation.py]: Possibly use the `no_reason` string here instead?

File: GearBot/Cogs/Moderation.py [hunk 3]
```diff
@@ -424,6 +425,26 @@ def _can_act(action, ctx, user, check_bot=True):
...
+                    else:
+                        reason = f"with reason `{reason}`"
```
Comment (trilleplay) [inline GearBot/Cogs/Moderation.py]: This should be a string so it can be translated

File: GearBot/Cogs/Moderation.py [hunk 4]
```diff
@@ -424,6 +425,26 @@ def _can_act(action, ctx, user, check_bot=True):
...
+                        GearbotLogging.log_to(ctx.guild.id, "verification_log", user=Utils.escape_markdown(ctx.author), user_id=ctx.author.id, level=level, reason=reason)
+                        await MessageUtils.send_to(ctx, 'YES', "verification_set", level=v1, reason=reason)
```
Comment (trilleplay) [inline GearBot/Cogs/Moderation.py]: these two lines probably shouldnt be in the `else:` aswell.

**Comment by Kitty-Therapist** [comment]
I went ahead and removed the unneeded return, although, I tried to test with "no_reason" string, and the bot doesn't appear to spit out the confirmation message. Gonna look into it when I'm more awake later unless AE is satisfied, along with the two lines being in else:

File: GearBot/Cogs/Moderation.py (hunk 2 above)
Comment (Kitty-Therapist) [reply GearBot/Cogs/Moderation.py]: Added!

File: GearBot/Cogs/Moderation.py (hunk 4 above)
Comment (Kitty-Therapist) [reply GearBot/Cogs/Moderation.py]: Fixed!

**Comment by Kitty-Therapist** [comment]
This PR should be ready to be merged, unless there are any further changes I need to make.

File: GearBot/Cogs/Moderation.py [hunk 5]
```diff
@@ -441,7 +441,7 @@ def _can_act(action, ctx, user, check_bot=True):
                     if reason == "":
                         reason = Translator.translate("no_reason", ctx.guild.id)
                     else:
-                        reason = f"with reason {reason}"
+                        reason = f"{reason}"
```
Comment (trilleplay) [inline GearBot/Cogs/Moderation.py]: 🤔 seems, kinda pointless

**Comment by trilleplay** [comment]
Alrighty most things look good, i'll let @AEnterprise do the approval review.

**Pull request merged**